src/java/voldemort/store/readonly/swapper/AdminStoreSwapper.java

```diff
@@ -146,6 +151,10 @@ public void invokeRollback(final String storeName, final long pushVersion) {
     public Map<Node, Response> invokeFetch(final String storeName,
                                            final String basePath,
                                            final long pushVersion) {
+        // This should result in 5 minutes max of wait time, for failure scenarios which are potentially recoverable.
+        final int MAX_FETCH_ATTEMPT = 10;
+        final long WAIT_TIME_BETWEEN_ATTEMPTS = 30000;
```

Comment:
Where will those two variables be used? I could not see it in this pull request.